packages/notebook/src/actions.tsx

```diff
@@ -1083,6 +1083,31 @@ namespace NotebookActions {
     Private.handleState(widget, state);
   }
 
+  /**
+   * Persists the collapsed state of all code cell outputs to the model.
+   *
+   * @param widget - The target notebook widget.
+   */
+  export
+  function persistOutputsCollapsed(widget: Notebook): void {
+    if (!widget.model || !widget.activeCell) {
```

Comment:
Do we need an active cell check here? As far as I know, we should always have one, but it doesn't seem strictly necessary for the functionality.

Reply:
Nope I don't believe we do, I was just copy and pasting. Removed.